modules/updateBookData.js

```diff
@@ -0,0 +1,11 @@
+export default function updateBookData(collectionData) {
+  localStorage.setItem('tasksList', JSON.stringify(collectionData));
+  const tasksList = JSON.parse(localStorage.getItem('tasksList'));
```

Comment:
- [ ]  Kindly check my comment on the use of es6 on the addToList file which is very detailed 👍